packages/daimo-expo-enclave/ios/SecureEnclaveKeyManager.swift

```diff
@@ -50,7 +50,7 @@ public class SecureEnclaveKeyManager : KeyManager {
         
         let signingPrivkey = try SecureEnclave.P256.Signing.PrivateKey(accessControl: accessControl)
         
-        try self.store.storeKey(signingPrivkey, account: accountName)
+        try self.store.storeKey(signingPrivkey, account: accountName, requireUserPresence: false)
```

Comment:
note this is set to false because otherwise user is prompted for biometrics twice (once to read the keychain data and once for Secure Enclave signing) which isn't great UX.

Reply:
LGTM but i'd put a short comment saying that ^

Reply:
done